paddle/fluid/framework/ir/mkldnn_placement_pass.cc

```diff
@@ -18,12 +18,22 @@ namespace paddle {
 namespace framework {
 namespace ir {
 
+
 std::unique_ptr<ir::Graph> MKLDNNPlacementPass::ApplyImpl(
     std::unique_ptr<ir::Graph> graph) const {
   VLOG(3) << "Aplies MKL-DNN placement strategy.";
+  const std::vector<std::string> op_list = 
```

Comment:
`const auto& op_list = Get<std::vector<std::string>>("mkldnn_op_list");`

Reply:
Will change it , thanks